src/libraries/System.Private.CoreLib/src/System/IO/RandomAccess.Windows.cs

```diff
@@ -679,6 +669,18 @@ private static unsafe ValueTask WriteFileGatherAsync(SafeFileHandle handle, Memo
             return new ValueTask(vts, vts.Version);
         }
 
+        private static async ValueTask WriteGatherAtOffsetMultipleSyscallsAsync(SafeFileHandle handle, IReadOnlyList<ReadOnlyMemory<byte>> buffers, long fileOffset, CancellationToken cancellationToken)
+        {
+            long bytesWritten = 0;
+            int buffersCount = buffers.Count;
+            for (int i = 0; i < buffersCount; i++)
+            {
+                ReadOnlyMemory<byte> rom = buffers[i];
+                await WriteAtOffsetAsync(handle, rom, fileOffset + bytesWritten, cancellationToken).ConfigureAwait(false);
+                bytesWritten += rom.Length;
+            }
+        }
```

Comment:
Did this code change?  I'm wondering why it moved. (Changes intended to be backported should involve as little change as possible.)

Reply:
I moved it for consistency with the `ReadScatter` methods. Feel free to bring it back on the PR to be backported.